pictureGenerator_HW/pictureGenerator_HW.js

```diff
@@ -0,0 +1,32 @@
+console.log("I'm here")
+
+
+
+
+const eventType = 'click'; //Type of event 
+
+
+
+const myButton = document.querySelector('button'); //Target 
+
+
+
+
+
+
+// Type + Target + Callback + Event Listener
+const imyButton = document.querySelector('#randomize'); //target
+
+
+const getnewElemnt = function() {
+    const Link = "https://source.unsplash.com/1600x900";
```

Comment:
you can make the the bonus by getting the height and width as a variable and using them as a part of the URL

```
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");
let url = ("https://source.unsplash.com/" + width + "x" + height);
let image = document.querySelector(".img");
image.setAttribute("src", url);
```